packages/headless/src/utils/highlight.test.ts

```diff
@@ -1,36 +1,101 @@
-import {HighlightKeyword, highlightString, HighlightParams} from './highlight';
+import {
```

Comment:
New test added starting at line 44. Added a `describe` wrapper for `highlightString` tests that's why the diff is "big".